packages/amplify-cli-core/src/utils/packageManager.ts

```diff
@@ -12,18 +12,18 @@ export type PackageManager = {
   executable: string;
 };
 
-const isWindows = (): boolean => /^win/.test(process.platform);
+const isWindows = process.platform === 'win32';
 
 const packageManagers: Record<string, PackageManager> = {
   npm: {
     packageManager: 'npm',
     lockFile: 'package-lock.json',
-    executable: isWindows() ? 'npm.cmd' : 'npm',
+    executable: isWindows ? 'npm.cmd' : 'npm',
```

Comment:
if we haven't already, we need to update the windows smoke tests with instructions for how to hit this codepath on windows